OHHTTPStubs/Sources/OHHTTPStubs+Mocktail.m

```diff
@@ -0,0 +1,125 @@
+//  Copyright (c) 2015 Capital One. All rights reserved.
+//
+
+#import "OHHTTPStubs+Mocktail.h"
+
+@class OHHTTPStubsDescriptor;
+
+@implementation OHHTTPStubs (Mocktail)
+
+
++(NSArray *)stubRequestsUsingMocktailsAt:(NSURL *)dirURL{
+    //make sure path points to a directory
+    BOOL isDir = NO, exists = NO;
+    NSError *error = nil;
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    exists = [fileManager fileExistsAtPath:dirURL.path isDirectory:&isDir];
+    if(!dirURL || !exists){
+        NSLog(@"Error opening %@: %@", dirURL.absoluteString, error);
+        return nil;
+    }
+    
+    //read the content of the directory
+    NSArray *fileURLs = [fileManager contentsOfDirectoryAtURL:dirURL includingPropertiesForKeys:nil options:0 error:&error];
+    if (error) {
+        NSLog(@"Error opening %@: %@", dirURL.absoluteString, error);
+        return nil;
+    }
+    
+    //stub the Mocktail-formatted requests
+    NSMutableArray *descriptorArray = [[NSMutableArray alloc] initWithCapacity:fileURLs.count];
+    for (NSURL *fileURL in fileURLs) {
+        if (![[fileURL absoluteString] hasSuffix:@".tail"]) {
+            continue;
+        }
+        id<OHHTTPStubsDescriptor> descriptor = [[self class] stubRequestsUsingMocktail:fileURL];
+        if(descriptor){
+            [descriptorArray addObject:descriptor];
+        }
+    }
+    
+    return descriptorArray;
+}
+
++(id<OHHTTPStubsDescriptor>)stubRequestsUsingMocktailNamed:(NSString *)fileName{
+    NSString *path = OHPathForFile(fileName, [self class]);
+    if(!path || ![[NSFileManager defaultManager] fileExistsAtPath:path]){
+        NSLog(@"File not exists at:%@", path);
+        return nil;
+    } else {
+        return [[self class] stubRequestsUsingMocktail:[NSURL fileURLWithPath:path]];
+    }
+}
+
++(id<OHHTTPStubsDescriptor>)stubRequestsUsingMocktail:(NSURL *)fileURL{
+    NSError *error = nil;
+    NSStringEncoding originalEncoding;
+    NSString *contentsOfFile = [NSString stringWithContentsOfURL:fileURL usedEncoding:&originalEncoding error:&error];
+    
+    if (!contentsOfFile || error) {
+        return nil;
+    }
+    
+    NSScanner *scanner = [NSScanner scannerWithString:contentsOfFile];
+    NSString *headerMatter = nil;
+    [scanner scanUpToString:@"\n\n" intoString:&headerMatter];
+    NSArray *lines = [headerMatter componentsSeparatedByString:@"\n"];
+    if ([lines count] < 4) {
+        if (error) {
+            error = [NSError errorWithDomain:@"Mocktail" code:0 userInfo:@{NSLocalizedDescriptionKey:[NSString stringWithFormat:@"Invalid amount of lines: %u", (unsigned)[lines count]]}];
+        }
+        return nil;
+    }
+    
+    /*handle Mocktail format, adapted from Mocktail implementation, for more details on the file format, check out: https://github.com/square/objc-Mocktail*/
+    NSRegularExpression *methodRegex = [NSRegularExpression regularExpressionWithPattern:lines[0] options:NSRegularExpressionCaseInsensitive error:nil];
```

Comment:
You shouldn't ignore the potential `error` here, especially since this `NSRegularExpression` is not hardcoded as a string literal but comes from a file so is highly prone to errors.